pycaching/trackable.py

```diff
@@ -17,7 +17,10 @@ def wrapper(*args, **kwargs):
             return func(*args, **kwargs)
         except AttributeError:
             logging.debug("Lazy loading: %s", func.__name__)
-            self.geocaching.load_trackable(self.tid, self)
+            try:
+                self.geocaching.load_trackable_by_url(self.trackable_page, self)
+            except AttributeError:
+                self.geocaching.load_trackable(self.tid, self)
```

Comment:
Wouldn't this cause an endless loop (recursion) in case of lazy loading `tid` of Trackable, which doesn't have its `trackable_page` specified? If yes, could you please handle it (some meaningful error message)?